recursive_art.py

```diff
@@ -1,152 +1,75 @@
-""" TODO: Put your header comment here """
+""" A file to generate random art with various algorithms """
 
 import random
-import numpy
+import numpy as np
```

Comment:
I think that there should be more involvement to get the actual performance benefits from numpy, rather than just calling with np.func instead of math.func. 

See [Oliver's solution](https://github.com/osteele/ComputationalArt/blob/numpy/recursive_art.py) where he generates all the pixels with numpy.